lib/pbench/server/api/__init__.py

```diff
@@ -87,7 +88,11 @@ def register_endpoints(api, app, config):
         f"{base_uri}/datasets/detail",
         resource_class_args=(config, app.logger),
     )
-
+    api.add_resource(
+        DatasetsPublish,
+        f"{base_uri}/datasets/publish",
+        resource_class_args=(config, app.logger),
```

Comment:
Of course I copied that from above, where the datasets endpoints also use `app.logger` ... I'll change all three. I suspect this crept in because you were doing some reorganization of the Flask infrastructure that merged while I was still working on the datasets queries and I missed this in the rebase.